src/main/java/org/apache/sysml/runtime/matrix/data/LibMatrixCUDA.java

```diff
@@ -175,6 +185,13 @@ private static cudnnFilterDescriptor allocateFilterDescriptor(int K, int C, int
 		return filterDesc;
 	}
 	
+	private static cudnnPoolingDescriptor allocatePoolingDescriptor(int R, int S, int pad_h, int pad_w, int stride_h, int stride_w) {
```

Comment:
Even though private methods typically don't have comments, it would really help for other contributors to understand.